packages/web/config/chain-infos.ts

```diff
@@ -630,6 +630,17 @@ const chainInfos = (
           coinGeckoId: "pool:muse",
           coinImageUrl: "/tokens/muse.svg",
         },
+        {
```

Comment:
Is a PR to the Chain Registry in the works?

Reply:
done
https://github.com/cosmos/chain-registry/pull/1056